linux/sound.cpp

```diff
@@ -37,18 +37,15 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
         serverName = "default";
     }
     qInfo() << qUtf8Printable (
-        QString ( "Connecting to jack \"%1\" instance (use the JACK_DEFAULT_SERVER environment variable to change this)." ).arg ( serverName ) );
+        QString ( "Connecting to JACK \"%1\" instance (use the JACK_DEFAULT_SERVER environment variable to change this)." ).arg ( serverName ) );
 
     // try to become a client of the JACK server
     pJackClient = jack_client_open ( jackClientName, JackNullOption, &JackStatus );
 
     if ( pJackClient == nullptr )
     {
-        throw CGenErr ( tr ( "The Jack server is not running. This software "
-                             "requires a Jack server to run. Normally if the Jack server is "
-                             "not running this software will automatically start the Jack server. "
-                             "It seems that this auto start has not worked. Try to start the Jack "
-                             "server manually." ) );
+        throw CGenErr ( tr ( "The JACK server couldn't be started automatically. "
```

Comment:
We refer to "JACK server" and "JACK" on its own here. It it easier to just to refer to "JACK" and not add the word "server"? Or are they two different things (in which case this message is confusing!)

Reply:
I think it’s just a technical description and probably there’s no need to talk about a server.

Reply:
```suggestion
        throw CGenErr ( tr ( "JACK couldn't be started automatically. "
```